test/TesterInternal/StorageTests/PersistenceGrainTests.cs

```diff
@@ -31,60 +31,56 @@ public class PersistenceGrainTests_Local : OrleansTestingBase, IClassFixture<Per
     {
         public class Fixture : BaseTestClusterFixture
         {
+
             protected override TestCluster CreateTestCluster()
             {
                 var options = new TestClusterOptions(initialSilosCount: 1);
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<MockStorageProvider>("test1");
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<MockStorageProvider>("test2", new Dictionary<string, string> { { "Config1", "1" }, { "Config2", "2" } });
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<ErrorInjectionStorageProvider>("ErrorInjector");
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<MockStorageProvider>("lowercase");
+                options.ClusterConfiguration.Globals.RegisterStorageProvider<MockStorageProvider>(MockStorageProviderName1);
+                options.ClusterConfiguration.Globals.RegisterStorageProvider<MockStorageProvider>(MockStorageProviderName2, new Dictionary<string, string> { { "Config1", "1" }, { "Config2", "2" } });
+                options.ClusterConfiguration.Globals.RegisterStorageProvider<ErrorInjectionStorageProvider>(MockStorageProviderNameErrorInjector);
```

Comment:
because I have longer naming convention .. lol
and to look prettier with other static provider names... 
 I can do ErrorInjectorProviderName :)